microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -3050,6 +3050,234 @@ Receiver:
 ---
 
 
+<!-- ====================================================================== -->
+## Web Notification (Incoming)
```

Comment:
where to put Web Notification APIs?

rendered incoming section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3253&tabs=dotnetcsharp#web-notification-incoming

outline at top of PR main page:
https://github.com/MicrosoftDocs/edge-developer/pull/3253

@nishitha-burman @peiche-jessica

Reply:
result (from top of PR main page - outline):
```
## Browser features
#### Printing
#### Cookies
#### Image capture
#### Downloads
#### Save as -- NEW SECTION
#### Web notification handling -- NEW SECTION
#### Permissions
#### Context menus
#### Status bar
#### Fluent overlay scrollbars
#### User Agent
#### Autofill
#### Audio
#### ...
```